src/journey-maps/angular/components/feature-event-listener/feature-event-listener.ts

```diff
@@ -69,6 +69,7 @@ export class SbbFeatureEventListener implements OnChanges, OnDestroy {
   private _mapCursorStyleEvent: SbbMapCursorStyleEvent;
   private _featuresHoverEvent: SbbFeaturesHoverEvent;
   private _featuresClickEvent: SbbFeaturesClickEvent;
+  private _listener: () => void;
```

Comment:
```suggestion
  private _stationListener: () => void;
```

or something like that so that it is evident where the listener belongs to.

Reply:
Done ✅